Automattic/wp-calypso
# Jetpack Backup: Add backup storage space component

#### Changes proposed in this Pull Request

This PR adds the backup storage space component as designed for the "Make backup and security products real-time only" project.

<img width="741" alt="image" src="https://user-images.githubusercontent.com/42627630/126551955-aaee5ea8-147a-4a15-acff-429b1264a603.png">

<img width="729" alt="image" src="https://user-images.githubusercontent.com/42627630/126552027-bfb86215-a9d1-497d-b823-4863c85353c4.png">

<img width="729" alt="image" src="https://user-images.githubusercontent.com/42627630/126552114-5c547a8a-49c0-4e90-a77a-8163986b00af.png">

<img width="723" alt="image" src="https://user-images.githubusercontent.com/42627630/126552175-90796c02-b1b1-40b4-9963-23e5cd808850.png">

#### Testing instructions



1. Visit `jetpack.cloud.localhost:3001/backup/{ site_slug }?flags=jetpack/only-realtime-products`
2. Edit the `usedStorage` variable in `client/my-sites/backup/backup-storage-space/index.tsx` to various values to test the different states shown in the screenshots above (10, 160, 185, and 200 covers all states).
3. Visit `jetpack.cloud.localhost:3001/backup/{ site_slug }` and verify that no storage space component displays.



Related to 1200412004370260-as-1200532973279659
